gist/src/main/java/build/gist/presentation/GistSdk.kt

```diff
@@ -86,6 +86,9 @@ object GistSdk : Application.ActivityLifecycleCallbacks {
                 Log.e(GIST_TAG, e.message, e)
             }
         }
+
+        // Initialising Gist web with an empty message to fetch all required assets.
```

Comment:
👍 Ah, this comment explains why this code was added. 

Although, I don't fully understand - What is "all required assets"? What is a required asset and why are they all needed to be fetched here on this line of code?